### PR TITLE
APP-330: Configuration Demo does not display seconds in the sampling period.

### PR DESCRIPTION
**Pull request by am1kk**
http://jira.kaaproject.org/browse/APP-330

### CODE REVIEW

**Comment by am1kk** [comment]
@Slash32 Review please.

**Comment by Slash32** [comment]
@kaa-leeroy test this please

**Comment by Slash32** [comment]
@kaa-leeroy test this fully please

File: configurationdemo/source/objc/ConfigurationDemo/ViewController.m [hunk 1]
```diff
@@ -106,7 +106,7 @@ - (void)onConfigurationUpdate:(KAAConfigurationConfiguration *)configuration {
 
 - (void)displayConfiguration {
     KAAConfigurationConfiguration *configuration = [self.kaaClient getConfiguration];
-    [self addLogWithText:[NSString stringWithFormat:@"Sampling period is %d", configuration.samplePeriod]];
+    [self addLogWithText:[NSString stringWithFormat:@"Sampling period is %d sec.", configuration.samplePeriod]];
```
Comment (Slash32) [inline configurationdemo/source/objc/ConfigurationDemo/ViewController.m]: Please folllow the rules used in the Bundle description. E.g.:

`[pool-2-thread-1] INFO  o.k.k.d.c.ConfigurationDemo - Sampling period is now set to 10 seconds`

**Review by Slash32 — CHANGES_REQUESTED** [review]

**Comment by am1kk** [comment]
@Slash32 re-review please.

**Comment by Slash32** [comment]
@kaa-leeroy test this fully please

**Review by Slash32 — APPROVED** [review]

**Review by Slash32 — APPROVED** [review]

**Pull request merged**